events/player-join-game.js

```diff
@@ -0,0 +1,36 @@
+const Utils = require('../utils');
+
+function checkIfGameExist(gameState) {
```

Comment:
This could easily be rewritten as:
``` js
function checkIfGameExists(gameState) {
  return gameState !== undefined;
}
```

But we'll leave it as is (it does the correct thing).